modules/targetAcquisition/targetAcquisitionWorker.py

```diff
@@ -10,7 +10,8 @@ def targetAcquisitionWorker(pause, exitRequest, pipelineIn, pipelineOut):
 
         curr_frame = pipelineIn.get()
```

Comment:
There is no guarantee that the pipeline contains valid data, especially if dummy data needs to be inserted for a stop request. Of course, if we're okay with the stop request being literally us killing the process, this is moot (but still, correct code is good code).

Reply:
I didn't consider the dummy data case, but this means we'll need to have validation once we pull data out, right? That's dependent on the shape of the data, including the dummy data. We need to define a) the shape of dummy data and b) use that to define the shape of valid data, and write a validator.